mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/modules/CassandraMailboxModule.java

```diff
@@ -50,6 +50,7 @@ public CassandraMailboxModule() {
                     .addColumn(CassandraMailboxTable.NAME, text())
                     .addColumn(CassandraMailboxTable.UIDVALIDITY, bigint())
                     .withOptions()
+                    .comment("Holds the mailboxes list.")
```

Comment:
a table is already a list. Is it a list of list ? I think it's rather a lookup table, right ?